### PR TITLE
First version

### PR DESCRIPTION
**Pull request by NaveenSidhartha**
Added podnet.py, robosoc.py, templates, .gitignore files

### CODE REVIEW

File: podnet.py [hunk 1]
```diff
@@ -0,0 +1,292 @@
...
+        else:
+            print("Invalid choice. Please enter '1' or '2'.")
+
+
+def display_eula():
+    """
+    Display the End User License Agreement (EULA).
+    Returns True if the user accepts, False otherwise.
+    """
+    eula_text = """
```
Comment (maria-walsh) [inline podnet.py:205]: Has this EULA been approved?

**Review by maria-walsh — COMMENTED** [review]
Please see comments

File: podnet.py (hunk 1 above)
Comment (NaveenSidhartha) [reply podnet.py:205]: Its just show off, will remove this because we using apache2 LICENSE file.

File: templates/nftables.nft.j2 [hunk 2]
```diff
@@ -0,0 +1,310 @@
...
+
+    # allow all packets sent by the Podnet box itself
+    chain output {
+        type filter hook output priority filter; policy accept;
+        # drop invalid traffic
+        ct state invalid drop
+    }
+
+}
+#!/usr/sbin/nft -f
```
Comment (maria-walsh) [inline templates/nftables.nft.j2:156]: Is this duplicate content for the rules?

File: templates/nftables.nft.j2 [hunk 3]
```diff
@@ -0,0 +1,310 @@
...
```
Comment (maria-walsh) [inline templates/nftables.nft.j2:1]: Have you discussed with Oscar how logging of the main firewall rules will be managed? I know it is been implemented in the current pods supported

File: templates/netplan.j2
Comment (maria-walsh) [inline templates/netplan.j2:1]: Is this template required? It is currently empty

File: templates/nftables.nft.j2 (hunk 2 above)
Comment (NaveenSidhartha) [reply templates/nftables.nft.j2:156]: After today's Scrum, I understood that Main firewall rules are applied by PAT (during adoption) which I guess a back end script(just like a driver) is triggered by any changes in firewall rules in database. This script applies the Main Firewall using `primitives.firewall` over OOB. So for PodNet installation step we don't need to configure Main Firewall rules. so removing it here.

File: templates/nftables.nft.j2 (hunk 3 above)
Comment (NaveenSidhartha) [reply templates/nftables.nft.j2:1]: same, no need this here

File: templates/netplan.j2
Comment (NaveenSidhartha) [reply templates/netplan.j2:1]: yes and removing it as well

File: podnet.py (hunk 1 above)
Comment (NaveenSidhartha) [reply podnet.py]: done

**Pull request merged**